pkg/kv/kvserver/batcheval/cmd_end_transaction.go

```diff
@@ -1036,6 +1036,9 @@ func splitTriggerHelper(
 	if !rec.ClusterSettings().Version.IsActive(ctx, clusterversion.VersionContainsEstimatesCounter) {
 		deltaPostSplitLeft.ContainsEstimates = 0
 	}
+	if !rec.ClusterSettings().Version.IsActive(ctx, clusterversion.VersionAbortSpanBytes) {
```

Comment:
Are you sure? If my reading of the callers of this code is correct, `deltaPostSplitLeft` will essentially be `ms`, meaning that the existing zeroing out should override it just fine. Don't you want to massage the `RHSDelta` above? That would make more sense to me as it is used to seed the stats for the RHS, and will not go through the existing check.